src/modules/sim-host/sim-status.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 Intel Corporation. All rights reserved.
```

Comment:
@busykai Sounds good. I emailed our legal team but we can modify it later if needed. Feel free to merge.